README.md

```diff
@@ -9,29 +9,48 @@
 [![Build Status](https://ci.appveyor.com/api/projects/status/github/donejs/donejs?branch=master&svg=true)](https://ci.appveyor.com/project/daffl/donejs)
 [![Coverage Status](https://coveralls.io/repos/github/donejs/donejs/badge.svg?branch=master)](https://coveralls.io/github/donejs/donejs?branch=master)
 
-DoneJS is the easiest way to get a high-performance, real-time, web and mobile application
-done! The framework provides a nearly comprehensive combination of technologies for
-building complex JavaScript applications.
+DoneJS is your new favorite way to get **high-performance**, **real-time**, **web and mobile** applications DONE!
 
-If you are looking for the fastest way to get a fully modern development environment setup
-and produce a lightning fast application, you’ve come to the right place.
+With a comprehensive set of packages designed specifically for complex JavaScript projects, and the ability to quickly
+provision a modern development environment, producing lightning-fast applications can now be done faster.
 
-DoneJS is a combination of the following technologies:
+The DoneJS environment automates the orchestration of these (otherwise disparate) parts:
 
-- [StealJS](http://stealjs.com) — ES6, CJS, and AMD module loader and builder
-- [CanJS](https://canjs.com) — Custom elements and model-view-viewmodel (MVVM) utilities
-- [jQuery](https://jquery.com/) — DOM helpers
-- [jQuery++](http://jquerypp.com) — Extended DOM helpers
-- [QUnit](https://qunitjs.com/) or [Mocha](https://mochajs.org/) — Assertion library
-- [FuncUnit](https://funcunit.com/) — Functional tests
-- [Testee](https://github.com/bitovi/testee) or [Karma](https://karma-runner.github.io/) — Test runner
-- [DocumentJS](http://documentjs.com) — Documentation
-- [can-ssr](https://github.com/canjs/ssr) — Server-side rendering utilities for CanJS
+| Package                 | Description |
+|:----------------------- |:----------- |
+| [StealJS][]             | ES6, CJS, and AMD module loader and builder
```

Comment:
GitHub support got back to me yesterday letting me know they were elevating this issue, but I haven't heard anything since then.